### ISSUE TITLE
CIFAR10 python version

### ISSUE THREAD

**Issue by HiPracheta** [avourakis/dataset-to-cifar#1]
Have you tried to convert the dataset into cifar10 python version? Can you share your code?

**Comment by avourakis** [comment]
> Have you tried to convert the dataset into cifar10 python version? Can you share your code?

@HiPracheta No, unfortunately this is still in my to-do list. Feel free to make improvements !
